src/js/build.js

```diff
@@ -2,12 +2,26 @@ import axios from 'axios'
 import JSZip from 'jszip'
 import Sass from 'sass.js/dist/sass'
 import CleanCSS from './lib/clean-css'
+import autoprefixer from './lib/autoprefixer'
 
 import { jsPlugins, scssPlugins } from './plugins'
 import { createJsFileContent, generateLink } from './file-util'
 import { formatScssList, uniqArray } from './util'
 
 const popperCDN = 'https://unpkg.com/popper.js/dist/umd/popper.js'
+const autoPrefixerBrowsers = [
```

Comment:
I'd change this to an object, keep the browsers as an array and add `cascade: false` like we do upstream.